pyfstat/grid_based_searches.py

```diff
@@ -101,14 +101,13 @@ def __init__(
             os.mkdir(outdir)
         self.set_out_file()
         self.search_keys = ["F0", "F1", "F2", "Alpha", "Delta"]
-        self.input_keys = ["minStartTime", "maxStartTime"] + self.search_keys
-        self.keys = self.input_keys.copy()
+        self.output_keys = self.search_keys.copy()
         if self.BSGL:
             self.detstat = "BSGL"
-            self.keys += ["logBSGL"]
+            self.output_keys += ["logBSGL"]
         else:
             self.detstat = "twoF"
-            self.keys += ["twoF"]
+            self.output_keys += [self.detstat]
```

Comment:
I love the "inconsistency" of "BSGL -> logBSGL" but "twoF" is just "twoF" :wink:

Reply:
Actually this makes sense: in the same notation as the BSGL family, F is essentially logBSG (log Bayes factor for Signal vs pure Gaussian noise). (Never mind the "two".)